include/onnxruntime/core/graph/graph.h

```diff
@@ -441,6 +443,13 @@ class Node {
     return attr_to_subgraph_map_;
   }
 
+  /** Gets a map of attribute name to the mutable Graph instances for all subgraphs of the Node.
+   * @returns a mutable map of mutable subgraphs.
+   */
+  std::unordered_map<std::string, gsl::not_null<Graph*>>& GetMutableAttributeNameToSubgraphMap() {
```

Comment:
I think it should be `GetMutableMapAttributeNameToSubgraph`. This is because keys are not mutable in maps and values are by default IF the map is mutable.

Reply:
Per offline discussion: `GetMutableMapOfAttributeNameToSubgraph`